ocaml/idl/datamodel.ml

```diff
@@ -2465,13 +2468,15 @@ let vm_assert_can_migrate = call
     ~name:"assert_can_migrate"
     ~in_product_since:rel_tampa
     ~doc:"Assert whether a VM can be migrated to the specified destination."
-    ~params:[
-      Ref _vm, "vm", "The VM";
-      Map(String,String), "dest", "The result of a VM.migrate_receive call.";
-      Bool, "live", "Live migration";
-      Map (Ref _vdi, Ref _sr), "vdi_map", "Map of source VDI to destination SR";
-      Map (Ref _vif, Ref _network), "vif_map", "Map of source VIF to destination network";
-      Map (String, String), "options", "Other parameters" ]
+    ~versioned_params:
+      [{param_type=Ref _vm; param_name="vm"; param_doc="The VM"; param_release=tampa_release; param_default=None};
+       {param_type=Map(String,String); param_name="dest"; param_doc="The result of a VM.migrate_receive call."; param_release=tampa_release;  param_default=None};
+       {param_type=Bool; param_name="live"; param_doc="Live migration"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vdi, Ref _sr); param_name="vdi_map"; param_doc="Map of source VDI to destination SR"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vif, Ref _network); param_name="vif_map"; param_doc="Map of source VIF to destination network"; param_release=tampa_release; param_default=None};
+       {param_type=Map (String, String); param_name="options"; param_doc="Other parameters"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vgpu, Ref _gpu_group); param_name="vgpu_map"; param_doc="Map of source vGPU to destination GPU group"; param_release=vgpu_tech_preview_release; param_default=Some (VMap [])}
```

Comment:
As above, this should be an upcoming release, not that old one.

Could this list of versioned_params be defined in a single place then shared between "migrate_send" and "assert_can_migrate"? (But that refactoring would be a separate commit, and possibly a separate pull-request etc.)